hack/generate-manifest-flow-visibility.sh

```diff
@@ -120,12 +182,21 @@ if [ "$MODE" == "e2e" ]; then
     cp $KUSTOMIZATION_DIR/base/kustomize-config.yml base/kustomize-config.yml
     cp $KUSTOMIZATION_DIR/base/provisioning/datasources/create_table.sh base/provisioning/datasources/create_table.sh
     cp $KUSTOMIZATION_DIR/../clickhouse-operator-install-bundle.yml clickhouse-operator-install-bundle.yml
+    cp $KUSTOMIZATION_DIR/patches/chmonitor/*.yml .
 
+    # patch the clickhouse monitor with desired storage size
     $KUSTOMIZE edit add base base
+    sed -i.bak -E "s/STORAGE_SIZE_VALUE/$SIZE/" chMonitor.yml
+    $KUSTOMIZE edit add patch --path chMonitor.yml --group clickhouse.altinity.com --version v1 --kind ClickHouseInstallation --name clickhouse
```

Comment:
@wsquan171 could you look at this. Do we need the monitor for e2e tests, that doesn't seem right to me?

Reply:
As previously we do not separate the monitor from the ClickHouse deployment, it is included by default in the e2e test. Thus I keep it in this update. But I'm glad to remove it if @wsquan171 thinks it is safe to do this.

Reply:
We don't really need or test the monitor now. As Yanjun mentioned it's included before since there's no harm to have it, but also as it's a hassle to just removed it for e2e. since now the manifest generation takes the monitor out by default I don't see the need to introduce more work to just add it back. we'd also want to mentioned that monitor is absent in the help message print when in e2e mode.

Reply:
Thanks! Removed the monitor related contents for the e2e mode.

Reply:
Thanks! Removed the monitor related contents for the e2e mode.